src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/util/RecommendationToPartADataMapperTest.kt

```diff
@@ -65,14 +65,24 @@ class RecommendationToPartADataMapperTest {
     }
   }
 
-  @ParameterizedTest(name = "given recall type {0}, recall details {1}, indeterminate sentence {2}, should map to the part A value {3} and details {4}")
-  @CsvSource("STANDARD,Standard details,false,Standard,Standard details", "FIXED_TERM,Fixed details,false,Fixed,Fixed details", "NO_RECALL,,false,,", "STANDARD,Standard details,true,N/A,N/A", "FIXED_TERM,Fixed details,true,N/A,N/A", "NO_RECALL,,true,N/A,N/A")
+  @ParameterizedTest(name = "given recall type {0}, recall details {1}, fixed term additional licence conditions {2}, indeterminate sentence {3}, extended sentence {4}, should map in the part A with the recall value {5} and details {6} with additional licence conditions value {7}")
+  @CsvSource(
+    "STANDARD,Standard details,,false,false,Standard,Standard details,N/A (standard recall)",
+    "FIXED_TERM,Fixed details,Fixed term additional licence conditions,false,false,Fixed,Fixed details,Fixed term additional licence conditions",
+    "NO_RECALL,,,false,false,,,",
```

Comment:
could maybe remove the no_recall case, it should never occur?

Reply:
done